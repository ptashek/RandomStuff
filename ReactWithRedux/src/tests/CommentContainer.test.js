import React from 'react';
import CommentContainer from '../components/CommentContainer.react';
import renderer from 'react-test-renderer';

import { TestComment } from './TestData';

it('renders ok', () => {
  const tree = renderer.create(
    <CommentContainer comment={TestComment} />
  ).toJSON();

  expect(tree.children.length).toBe(2);
  expect(tree).toMatchSnapshot();
});
