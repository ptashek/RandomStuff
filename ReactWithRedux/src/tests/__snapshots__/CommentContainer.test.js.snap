exports[`test renders ok 1`] = `
<div>
  <div
    className="Comment-Body">
    body
  </div>
  <div
    className="Comment-Footer">
    by 
    name
     (
    name@example.org
    )
  </div>
</div>
`;
